docs/docs/copilot.md

```diff
@@ -0,0 +1,84 @@
+---
+id: tooljet-copilot
+title: Copilot
+---
+
+:::info BETA
+ToolJet Copilot is currently available in public beta for all the users.
+:::
+
+**ToolJet Copilot** helps you write your queries faster. It uses OpenAI to suggest queries based on your data. 
+
+## Activating Copilot
+
+To access the Copilot, each user, including administrators, can sign up for the waitlist program individually. After signing up, each user will receive a unique API key associated with their account. **Only administrators**, however, will have the privilege to enable Copilot within the workspace settings. Administrators can navigate to the Copilot section in the workspace settings and toggle on the "Enable Copilot" option using their respective API key.
+
+You can join the waitlist here: **https://tooljet.com/copilot**
+
+:::info
+- Copilot can be used by users who have **permissions to edit** the app with the copilot setup.
+:::
+
+<div style={{textAlign: 'center'}}>
+
+<img className="screenshot-full" src="/img/copilot/enable.png" alt="ToolJet Copilot" />
+
+</div>
+
+## Using Copilot
+
+Once activated, the Copilot feature can be accessed while editing any query within the transformations section in the query editor.
+
+For Copilot to operate, it requires a plain English prompt. Copilot possesses the ability to interact with and access components, along with generating code for specific actions related to those components. This capability is made possible by Copilot's awareness of the application's current state.
```

Comment:
Copilot's functionality relies on the provision of clear and concise plain English prompts. It possesses the capability to interact with and retrieve information from various components, enabling it to generate code specific to the desired actions associated with those components. This capability is facilitated by Copilot's comprehensive understanding of the application's present state.